File: app/src/main/java/com/example/rahmatantravel/KiblatActivity.java
package com.example.rahmatantravel;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class KiblatActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_kiblat);

        ImageView backToMenuLainnya = findViewById(R.id.backToMenuLainnya);

        backToMenuLainnya.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });
    }
}

File: app/src/main/java/com/example/rahmatantravel/KiblatActivity.kt
package com.example.rahmatantravel;

import android.content.Intent
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import androidx.cardview.widget.CardView

class KiblatActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_kiblat)

        val backToMenuLainnya = findViewById<ImageView>(R.id.backToMenuLainnya)
        val buttonLanjut = findViewById<CardView>(R.id.buttonLanjut)

        buttonLanjut.setOnClickListener {

            val intent = Intent(this, KiblatActivity2::class.java)
            startActivity(intent)
        }

        backToMenuLainnya.setOnClickListener {
            onBackPressed()
        }
    }
}
